### DISCUSSION TITLE
Feedback para practica Abogabot

### DISCUSSION THREAD

**Discussion by CharlieCCR** [Launch-X-Latam/MisionFrontEnd#220] [Show and tell]
¡Qué tal a todos!
Me encantaría saber si me pudiesen dar sus opiniones y o recomendaciones con respecto a la practica de Abogabot. De momento es todo lo que tengo y con su feedback podría ir avanzando para realizar lo que sería las practicas HTML. 
Me gustaría aclarar que, de momento enfoque la idea de Abogabot para que fuese una aplicación de celular, pero me percate que muchos de mis otros compañeros explorers están realizando esta practica pero enfocada al diseño de una pagina web, es por ello que, igualmente me gustaría saber si ¿Con este diseño que realice, aunque este hecho para una aplicación móvil, a la hora de hacer la practica HTML podría pasar todo este diseño a un formato de pagina web, o tendría que re-hacer el diseño y pasarlo a un formato de pagina web? 
¡Saludos a todos! 
https://github.com/CharlieCCR/Caso-Abogabot

**Comment by doguedogue** [comment]
Se ve muy bien 🚀🚀🚀

Si renombras el archivo a README.md se abrirá automaticamente al abrir la dirección de tu repositorio, es algo asi como el index.html en la web.

**Comment by BrayanGil090720** [comment]
que tal compañero, le recomiendo hacer unas flechas guias para cada boton, es decir, si le pico aqui me redirige a tal lado, si le pico aqui me redirecciona a otro lado y asi 
![image](https://user-images.githubusercontent.com/114033789/197078799-8ed4a56d-dd89-4663-888b-d1abeff87f28.png)

**Reply by CharlieCCR** [reply]
Excelente, mil gracias compañero, de hecho si le había puesto flechas jaja, nada más que creo que subí el archivo equivocado, pero gracias por la observación de todos modos. 😅
